### ISSUE TITLE
Fix cause of failing ios build in CI

### ISSUE THREAD

**Issue by nickmeinhold** [adventuresin/chat_app#96]
I added an android build to the ios workflow at some point as I was going to ask if people were interested in having just a single workflow that runs on macOS and builds, tests and deploys both iOS and Android. Now we are getting the error: 

```
 No version of NDK matched the requested version 21.0.6113669. Versions available locally: 18.1.5063045, 21.1.6352462, 21.1.6352462
```

see [Add notifications (#85) · adventuresin/chat_app@cbf472b](https://github.com/adventuresin/chat_app/runs/638389118?check_suite_focus=true)

The latest NDK on macOS is older than the one on the linux workflow, hence the error. 

I am still interested in having that conversation as I think there is a lot of benefit to a single workflow but I will just remove the Android build in the iOS workflow for now to get us back to green.

**Comment by Jei** [comment]
One of the NDK versions listed in the error as locally available (21.1.6352462) is the latest. I've tried to download it and update the `ndkVersion` configuration in `build.gradle` and it seems to be working fine. Before returning to separate workflows, I think we could try a PR with that change. I'll put it up now.

**Comment by nickmeinhold** [comment]
Oh man, helps if you read the error message properly hey!  I had the same error a while back on another project and I guess I just pattern matched and assumed it was the same thing.  The latest NDK at the time was pretty old. Apparently it was recently [updated](https://github.com/actions/virtual-environments/issues/120). Very good to know!
